tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -1804,20 +1804,28 @@ public function checkIfShareExists(string $share, string $sharee, string $sharer
 				$foundShareInSharedWithMe = true;
 			}
 		}
-		Assert::assertTrue($foundShareInSharedWithMe, "Share '$share' was not found in the shared-with-me list");
+
+		if ($shouldOrNot === "should not") {
+			Assert::assertFalse($foundShareMountpoint, "Expected share mountpoint '$share' not to be present in the drives list but it was found.");
+			Assert::assertFalse($foundShareInSharedWithMe, "Expected '$share' not to be present in the shared-with-me list but it was found.");
+		} else {
+			Assert::assertTrue($foundShareMountpoint, "Expected share mountpoint '$share' to be present in the drives list but it was not found.");
+			Assert::assertTrue($foundShareInSharedWithMe, "Expected '$share' to be present in the shared-with-me list but it was not found.");
+		}
 	}
 
 	/**
-	 * @Then user :sharee should have a share :share shared by user :sharer
+	 * @Then /^user "([^"]*)" (should|should not) have a share "([^"]*)" shared by user "([^"]*)"$/
 	 *
 	 * @param string $sharee
+	 * @param string $shouldOrNot
 	 * @param string $share
 	 * @param string $sharer
 	 *
 	 * @return void
 	 */
-	public function userShouldHaveShare(string $sharee, string $share, string $sharer): void {
-		$this->checkIfShareExists($share, $sharee, $sharer);
+	public function userShouldHaveShare(string $sharee, string $shouldOrNot, string $share, string $sharer): void {
```

Comment:
```suggestion
	public function userShouldOrShouldNotHaveShare(string $sharee, string $shouldOrNot, string $share, string $sharer): void {
```

Reply:
Removed this method as another existing scenario used. :smile_cat: